.pylintrc

```diff
@@ -13,6 +13,8 @@ persistent=yes
 # usually to register additional checkers.
 load-plugins=
 
+# Whitelist pydantic extension
+extension-pkg-allow-list=pydantic
```

Comment:
This is the reason pydantic had to be added here: https://github.com/pydantic/pydantic/issues/1961#issuecomment-759522422